L59-core-allow-cppstdlib.md

```diff
@@ -0,0 +1,122 @@
+Allow C++ standard library in gRPC Core Library
```

Comment:
I did another pass on this document and I was able to get the info I needed (I think), but overall I felt the readability of the doc could be improved. Since the gRFCs are basically a part of official gRPC documentation, I'd recommend getting a language review before merging the final version of the doc.

Reply:
I really appreciate your thorough review! I'll reach out to the language reviewer.